src/style.css

```diff
@@ -0,0 +1,98 @@
+/*
+  intrinsic: 348 x 348
+  size: 250 x 250
+} */
+
```

Comment:
I assume all your css will be in one file? Mine are imported separately, maybe I should reconsider.

Reply:
Yea, my webpack bundles everything together.